pimd/pim_cmd.c

```diff
@@ -6596,82 +6510,6 @@ DEFUN (show_ip_ssmpingd,
 	return CMD_SUCCESS;
 }
 
-static int pim_rp_cmd_worker(struct pim_instance *pim, struct vty *vty,
-			     const char *rp, const char *group,
-			     const char *plist)
-{
-	int result;
-
-	result = pim_rp_new_config(pim, rp, group, plist);
-
-	if (result == PIM_GROUP_BAD_ADDR_MASK_COMBO) {
-		vty_out(vty, "%% Inconsistent address and mask: %s\n",
-			group);
-		return CMD_WARNING_CONFIG_FAILED;
-	}
-
-	if (result == PIM_GROUP_BAD_ADDRESS) {
-		vty_out(vty, "%% Bad group address specified: %s\n", group);
-		return CMD_WARNING_CONFIG_FAILED;
-	}
-
-	if (result == PIM_RP_BAD_ADDRESS) {
-		vty_out(vty, "%% Bad RP address specified: %s\n", rp);
-		return CMD_WARNING_CONFIG_FAILED;
-	}
-
-	if (result == PIM_RP_NO_PATH) {
-		vty_out(vty, "%% No Path to RP address specified: %s\n", rp);
-		return CMD_WARNING;
-	}
-
-	if (result == PIM_GROUP_OVERLAP) {
```

Comment:
This error message appears to have been lost:

old:
eva(config)# ip pim rp 4.5.6.7 224.1.0.0/16
eva(config)# ip pim rp 192.168.161.33 224.1.0.0/16
% Group range specified cannot exact match another

new:
eva(config-if)# ip pim rp 4.5.6.7 224.1.0.0/16
eva(config)# ip pim rp 192.168.161.33 224.1.0.0/16
eva(config)# do show run
Building configuration...

Current configuration:
ip pim rp 4.5.6.7 224.1.0.0/16